NEWS

```diff
@@ -1,3 +1,20 @@
+17-May-2018 IGNITION v0.25.0
+
+  Features:
+
+     - Blackbox test can now be run in parallel
+
+  Changes:
+
+     - Remove Oracle Cloud Infrastructure support
+
+  Bug Fixes:
+
+     - Fix issue where Ignition would leak a temporary file when appending to
```

Comment:
This could be read as leaking a file in `/tmp`.  Maybe we should say that it left a temporary file in the same directory as the existing one?

Reply:
`would leave a stray file`?  `leak` is a bit jargony here.